### ISSUE TITLE
Version update with new updated conda dependencies

### ISSUE THREAD

**Issue by tsenapathi** [protocaller/ProtoCaller#21]
Hello @msuruzhon, We have updated some package dependencies for protocaller in conda-forge. 
1. PDB2PQR (https://anaconda.org/conda-forge/pdb2pqr)- This version dropped the PDB2PKA/APBS support from within PDB2PQR  (temporarily) to get the independent conda install. 
2. pypdb (https://anaconda.org/conda-forge/pypdb)
3. propka (https://anaconda.org/conda-forge/propka)
And some more...

Especially we need to change htmd-pdb2pqr to the officially supported pdb2pqr package. So can we update the dependencies, run the tests and release a new version?

**Comment by msuruzhon** [comment]
Hi @tsenapathi, this is very good! Since `ProtoCaller` doesn't depend on `pypdb` anymore, this is no longer a requirement, but changing `htmd-pdb2pqr` to `pdb2pqr` requires some work. The tests for the tentative `1.2.0` version seem to pass now on my own laptop, but strangely enough I can't get the build to work, so there is no `conda` package yet. I am not exactly sure what is going wrong, I will try to take a look in the next few days, but it might be worth trying to relax the `pdb2pqr` requirements, as they seem to be clashing with the other dependencies.

**Comment by tsenapathi** [comment]
@msuruzhon Great!! Let me know if there is anything I can do to assist.

**Comment by msuruzhon** [comment]
The `conda` package is up and running and I have committed to `master`. I have also created two releases: `1.1.2`, which covers the changes since the overhaul of the Protein Data Bank, and `1.2.0`, where the only change relative to `1.1.2` is the `pdb2pqr` dependency. Both of them should be installable from `conda` and both of them should be functional. Feel free to test them and reopen if there are any issues.
